### ISSUE TITLE
经常性出现这个问题 是什么原因。我开了代理。

### ISSUE THREAD

**Issue by ghostrunner-art** [RSSNext/Follow#1240]
### Describe the bug

Feed 获取错误: [GET] "http://10.1.32.41/bilibili/user/video/89619004?format=json": 503 Service Unavailable 

### Feed Info

@sky00145

### Environment

**App Version**: 0.1.1-beta.1
**OS**: Windows
**User Agent**: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Follow/0.1.1-beta.1 Chrome/130.0.6723.44 Electron/33.0.0 Safari/537.36
**Env**: electron
**Browser**: Chrome

### App Type

Electron / App

### Version

**App 0.1.1-beta.1Version**: 0.1.1-beta.1

### Operating System

Windows 11

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by Innei** [comment]
Here's an example of rsshub getting a bililibi timeout. @pseudoyu

**Comment by pseudoyu** [comment]
Yes the official Follow RSSHub instance has a high volume of requests and frequently triggers anti-crawler mechanisms. We are working on it.

**Comment by north82** [comment]
yes, i have same problem. In my follow list, i see 2-3 error since a day among 12 followers
